.github/workflows/compute-impacted-libraries.yml

```diff
@@ -91,7 +91,7 @@ jobs:
               "version": "dev",
             }
             for library in sorted(result)
-            if "otel" not in library and library not in ("cpp_nginx", )
+            if "otel" not in library and library not in ("cpp_httpd", "cpp_nginx")
```

Comment:
it means that your removed `dev` CI  for `cpp_httpd`, is it what you want? If yes, what is needed to put it back ?